README.md

```diff
@@ -1 +1,64 @@
-# Team10-League-of-Legends
\ No newline at end of file
+# Team10-League-of-Legends
+
+[Wireframe](#wireFrame)
+[Design](#design)
+[Description](#description)
+[Build with](#buildWith)
+[Features](#features)
+[User journey](#userJourney)
+[User stories](#userStories)
+[Live link](#liveLink)
+[Mentor](#mentor)
+[Team members](#teamMember)
+# Wire Frame <span id="wireFrame"> </span>
+
+![](image/WireFrame.jpeg)
+# Design <span id="design"> </span>
+## Mobile 
+![](image/Phone.png)
+## Desktop 
+![](image/Desktop.png)
+
+# Description <span id="description"> </span>
+Our website is the perfect website for League of Legends players, it provides them with information about League of Legends's champions and displays its available skins
+
+
+## Features <span id="features"> </span>
+- Animation
+- API
+- Responsive 
+- Simple UI
+# Build with <span id="buildWith"> </span>
+- HTML
+- CSS
+- JavaScript
+- Local Storage
+
+# User Journey <span id="userJourney"> </span>
+When the user open the website loading page will show up after a while will disappear 
+the slider page will present four images at the end it move you to the home page
+the home page include:
+1- Search bar
+2- Characters pictures {
+    contains :
+    skin button: show up a page contains the skins of the champion.
+    details button: show up a page contains information about the champion.}
+3- Move buttons
+
```

Comment:
that's not the correct way to write a user journey, it should include the steps that the user will take from the landing page till the last one (it can be written as points)